vmdb/lib/miq_automation_engine/engine/miq_ae_workspace.rb

```diff
@@ -151,10 +149,10 @@ def instantiate(uri, root=nil)
         $miq_ae_logger.error(err.message)
       rescue MiqAeException::AssertionFailure => err
         $miq_ae_logger.info(err.message)
-        @graph.delete(v)
+        delete(obj)
```

Comment:
No, it doesn't. [But the old delete method didn't do that either](https://github.com/ManageIQ/manageiq/commit/2ef1fed#diff-387f53928fe366a9010046c8543ba5b7R38).